azext_iot/central/models/devicetwin.py

```diff
@@ -11,6 +11,8 @@ def __init__(
         device_twin: dict,
     ):
         self.device_twin = device_twin
+        if "_links" in device_twin:
```

Comment:
Curious what `_links` represents?

Reply:
this is an internal API used by other Central services as well. It usually contains the links to self with our webapi syntax. Nothing secret but worth to hide from the output as unuseful.

Reply:
Considering that this affects digital twins in general, it can affect other new features that may touch _links. We may need to unhide this in the future.

Reply:
Just want to point out we are talking around device twins, rather than digital twins.